File: python/iniciantes/1008.py
#1008 SALÁRIO
numeroFuncionario = int(input())
horasTrabalhadas = int(input())
valorPorHora = float(input())

calculo = horasTrabalhadas*valorPorHora
print("NUMBER =",numeroFuncionario)
print("SALARY = U$ {:.2F}" .format(calculo))

File: javascript/1008.js
/*Salário*/
let input = require("fs").readFileSync("/dev/stdin", "utf8");

const lines = input.split("\n");

const Funcionario = parseFloat(lines.shift());
const Horas = parseFloat(lines.shift());
const ValorporHora = parseFloat(lines.shift());

const salario = Horas * ValorporHora;

console.log("NUMBER = " + Funcionario);
console.log("SALARY = U$ " + salario.toFixed(2));